src/containers/Home.css

```diff
@@ -0,0 +1,3 @@
+div {
+  text-align: center;
```

Comment:
very generic centering, this will apply to all divs in the Application, best use a class